src/utils/transaction.js

```diff
@@ -134,52 +134,35 @@ function transferInt() {
     var oldBytes = convert_1.default.stringToByteArray('40004');
```

Comment:
Use `stringToByteArray` is wrong. Please add `int32ToByteArray` in `convert.js` and use this method to convert ByteArray.